### PR TITLE
SwiftLayoutPrinter has flags not LayoutOptions

### PR DESCRIPTION
**Pull request by oozoofrog**

### CODE REVIEW

**Review by gmlwhdtjd — COMMENTED** [review]
How about create a LayoutPrintingOptions separately from the LayoutOptions?

**Comment by oozoofrog** [comment]
> How about create a LayoutPrintingOptions separately from the LayoutOptions?

very agree

File: Sources/SwiftLayout/Util/IdentifierUpdater.swift [hunk 1]
```diff
@@ -9,30 +9,26 @@ import Foundation
 import UIKit
 
 public final class IdentifierUpdater {
+    /// update identifiers with property name
+    public static var nameOnly: IdentifierUpdater { IdentifierUpdater(.nameOnly) }
```
Comment (oozoofrog) [inline Sources/SwiftLayout/Util/IdentifierUpdater.swift:13]: @gmlwhdtjd  how about this? and thanx for suggestion that options thing.

**Review by gmlwhdtjd — COMMENTED** [review]
👍

**Pull request merged**